src/App.js

```diff
@@ -1,27 +1,15 @@
-import logo from './logo.svg';
+/* eslint-disable import/no-named-as-default-member */
+/* eslint-disable import/no-named-as-default */
+/* eslint-disable import/no-cycle */
+// import React, {Component} from 'react';
+// import logo from './logo.svg';
```

Comment:
- [ ] @Moise-code it's not best practice to disable the linter check in a file, kindly fix the linter errors instead of disabling them.